Makefile

```diff
@@ -1,16 +1,25 @@
+BIN_DIR:=node_modules/.bin
+
 compile:
 	./rebar3 compile
 
-dev: compile bower webpack
-	./rebar3 shell
+dev: compile webpack
+	./rebar3 shell --apps xprof
```

Comment:
- what about always setting dev profile
- what about always starting sync too? (useful in dev mode)

` export REBAR_PROFILE=dev ./rebar3 shell --apps xprof sync`